VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/api/VTCareResource.java

```diff
@@ -80,4 +82,22 @@ public Response login(String loginInfo) throws Exception {
             throw new Exception("An exception occurred while logging in.");
         }
     }
+
+    /**
+     *
+     * @return the providers
+     * @throws Exception if cascaded from DAO.
+     */
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("providers")
+    public List<Provider> getProvider() throws Exception {
```

Comment:
May change the name to getProvider**s**() like at rest of the places since it is returning a list. Rest all looks good.